src/noisepy/seis/noise_module.py

```diff
@@ -263,8 +266,8 @@ def preprocess_raw(st, inv, prepro_para, date_info):
                     print("removing response for %s using inv" % st[0])
                     st[0].attach_response(inv)
                     st[0].remove_response(output=rm_resp_out, pre_filt=pre_filt, water_level=60)
-                except Exception:
-                    print("WARNING: Failed to remove response from %s. Returning empty stream." % st[0])
+                except Exception as e:
+                    print("WARNING: Failed to remove response from %s. Returning empty stream. %s" % (st[0], e))
```

Comment:
@mdenolle if I test with `rmresp="inv"`, using the `SCEDCS3DataStore` (S3 data), this fails with the error:
```
Can not use evalresp on response with no response stages.
```
In this case the inventory is obtained using:
```
         client = Client(self.url_key)
         inventory = client.get_stations_bulk(bulk_station_request, level="channel")
```
Maybe this is not a valid configuration to use? I think this should be using the files from: s3://scedc-pds/FDSNstationXML/? These are all .XML files, e.g. (`CI_ABL.XML`). Is this what I should use? And which type of `rm_resp` does this correspond to? (`spectrum|RESP|polozeros`)

Reply:
This has been resolved in https://github.com/mdenolle/NoisePy/pull/97